README.md

```diff
@@ -365,6 +364,8 @@ The decision node is not fully functional as there is not currently
 support for all EL functions. So in order for it to run in Airflow you must
```

Comment:
```suggestion
 support for all EL functions. So in order for it to run in Airflow you may need to
```

I think that decision node implemented in #326 should handle most of the simple predicates.